### DISCUSSION TITLE
How can I exclude data for certain tables but keep structure with mysqldump?

### DISCUSSION THREAD

**Discussion by aveein** [spatie/db-dumper#212] [Q&A] [answered]
I'm doing a regular dump of a database  and excludeTables tends to remove tables structure too but i only want to remove it's data but keep the table structure for example i want to dumb database with log table but not it's data

**Comment by poldixd** [comment]
Have you found a solution yet?

**Reply by aveein** [reply, marked as answer]
First, obtain the skeleton of the excluded table, merge it with the included table, and then zip the sql.
